docs/_data/whys.yml

```diff
@@ -10,7 +10,8 @@
   icon: globe
```

Comment:
While we're here, we might as well fix the missing space on line 8 ("imagery,terrain").